components/grid/export/pdf.md

```diff
@@ -10,17 +10,223 @@ position: 1
+             Data="@GridData"
+             FilterMode="@GridFilterMode.FilterMenu"
+             Pageable="true"
+             Sortable="true"
+             Reorderable="true"
+             Resizable="true"
+             Groupable="true"
+             Width="650px">
+
+    <GridToolBarTemplate>
+        <GridCommandButton Command="PdfExport" Icon="@SvgIcon.FilePdf">Export to PDF</GridCommandButton>
+        <label class="k-checkbox-label"><TelerikCheckBox @bind-Value="@ExportAllPages" />Export All Pages</label>
+    </GridToolBarTemplate>
+
+    <GridExport>
+        <GridPdfExport FileName="telerik-grid-export" AllPages="@ExportAllPages" />
+    </GridExport>
+
+    <GridColumns>
+        <GridColumn Field="@nameof(SampleData.ProductId)" Title="ID" Width="100px" />
+        <GridColumn Field="@nameof(SampleData.ProductName)" Title="Product Name" Width="200px" />
+        <GridColumn Field="@nameof(SampleData.UnitsInStock)" Title="In stock" Width="100px" />
+        <GridColumn Field="@nameof(SampleData.Price)" Title="Unit Price" Width="100px" />
+        <GridColumn Field="@nameof(SampleData.Discontinued)" Title="Discontinued" Width="150px" />
+    </GridColumns>
+</TelerikGrid>
+
+@code {
+    private TelerikGrid<SampleData> GridRef { get; set; }
+
+    private MemoryStream exportedPdfStream { get; set; }
+
+    private List<SampleData> GridData { get; set; }
+
+    private bool ExportAllPages { get; set; }
+
+    private async Task GetTheDataAsAStream()
+    {
+        MemoryStream finalizedStream = await GridRef.ExportToPdfAsync();
+
+        exportedPdfStream = new MemoryStream(finalizedStream.ToArray());
+    }
+
+    protected override void OnInitialized()
+    {
+        GridData = Enumerable.Range(1, 100).Select(x => new SampleData
+            {
+                ProductId = x,
+                ProductName = $"Product {x}",
+                UnitsInStock = x * 2,
+                Price = 3.14159m * x,
+                Discontinued = x % 4 == 0,
+            }).ToList();
+    }
+
+    public class SampleData
+    {
+        public int ProductId { get; set; }
+        public string ProductName { get; set; }
+        public int UnitsInStock { get; set; }
+        public decimal Price { get; set; }
+        public bool Discontinued { get; set; }
+    }
+}
+````
+
+## Customization
+
+To customize the exported file, handle the `OnBeforeExport` or `OnAfterExport` events the Grid exposes. 
+
+The component allows you to control the data set that will be exported. It also provides built-in customization options for the columns such as `Width`, `Title`, and more.
+
+For more advanced customization the Grid lets you get the `MemoryStream` of the file. Thus, you can customize it using the [`PdfProcessing`](https://docs.telerik.com/devtools/document-processing/libraries/radpdfprocessing/overview) library that is available with your license.
+
+Read more about how to [customize the exported file](slug:grid-export-events).
+
+## Notes
```

Comment:
Notes sections tend to contains unstructured and unfindable information, and don't scale well at all. Consider:

* Move some of these to the Basics section, for example the info about column widths, as it's important.
* User H3 headings in Notes or somewhere else to provide granular and linkable structured information.

Reply:
Will be handled as part of https://github.com/telerik/blazor/issues/10895.